jcg_js_callgraph_testadapter/src/main/scala/JSCallGraphAdapter.scala

```diff
@@ -1,6 +1,6 @@
 import java.io.File
 
-object JSCallGraphAdapter extends TestAdapter {
+object JSCallGraphAdapter extends JSTestAdapter {
```

Comment:
Is the main method in this object still needed?

Reply:
It makes it possible to easily execute only the testadapter on the testcases and nothing else, which I mostly used for development and testing, so I would favor keeping it!